ui/src/overridden/frontsite/DocumentRequest/DocumentRequestFormHeader.js

```diff
@@ -6,9 +6,9 @@ import { Link } from "react-router-dom";
 export const DocumentRequestFormHeader = () => {
   return (
     <>
-      <Header as="h1">Request new document</Header>
+      <Header as="h1">Request document</Header>
       <p>
-        Fill in the form below to request to loan, purchase or suggest a new document
+        Fill in the form below to request to loan, purchase or suggest a document (book,
```

Comment:
Might have missed removing this extra `(book,`?

Reply:
Prettier's fault.. fixed now!